sample/host/src/bootstrap.tsx

```diff
@@ -2,12 +2,19 @@ import { ConsoleLogger, Runtime, RuntimeContext, registerLocalModules } from "@s
 import { StrictMode, Suspense } from "react";
 
 import { App } from "./App.tsx";
-import type { AppContext } from "@sample/shared";
+import { type AppContext, isNetlify } from "@sample/shared";
 import { createRoot } from "react-dom/client";
 import { register } from "@sample/local-module";
-import { registerRemoteModules } from "@squide/webpack-module-federation";
+import { type RemoteDefinition, registerRemoteModules } from "@squide/webpack-module-federation";
 import { sessionAccessor } from "./session.ts";
 
+const Remotes: RemoteDefinition[] = [
+    {
+        name: "remote1",
+        url: isNetlify ? "https://squide-remote-module.netlify.app" : "http://localhost:8081"
```

Comment:
this should be done via a .env file. I feel like we don't want utils methods like "isNetlify" in code that are used as samples.

Reply:
I don't share both of your concerns here.